pkg/apis/batch/v2alpha1/types.go

```diff
@@ -250,6 +250,16 @@ type CronJobSpec struct {
 	// JobTemplate is the object that describes the job that will be created when
 	// executing a CronJob.
 	JobTemplate JobTemplateSpec `json:"jobTemplate" protobuf:"bytes,5,opt,name=jobTemplate"`
+
+	// The number of successful finished jobs to retain.
+	// This is a pointer to distinguish between explicit zero and not specified.
+	// +optional
+	SuccessfulJobsHistoryLimit *int32 `json:"successfulJobsHistoryLimit,omitempty" protobuf:"varint,6,opt,name=successfulJobsHistoryLimit"`
```

Comment:
The other options is, since we're introducing `batch/v1beta1.CronJob` we could introduce these defaults there, and let the `batch/v2alpha1.CronJob` just introduce these fields, but without defaulting, wdyt?